### PR TITLE
[Term Entry] Pillow fromarray

### LINKED ISSUE #4624: [Term Entry] Python pillow .fromarray()
### Reason/inspiration (optional)

We would like a new term entry on the '.fromarray()' method for the `Pillow` library in Python. The entry should go in a new file under `docs/content/pillow/concepts/image/terms/fromarray/fromarray.md`

The entry should include the following:

- A description of the function
- A ## Syntax section that includes the syntax of the function and its details
- An ## Example section that includes a ```python ``` codeblock and ```shell ``` that shows the example output.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by deenovita**
### Description

I have added a new term entry for the .fromarray function from Pillow Library

### Issue Solved

Ask to close #4624 issue

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pillow/concepts/image/terms/fromarray/fromarray.md
Comment (SaviDahegaonkar) [inline content/pillow/concepts/image/terms/fromarray/fromarray.md:51]:
```suggestion
from PIL import Image
import numpy as np

# Defined an array 3 by 3 with 200 as all the pixel values
obj_array = np.full((3, 3), 200)

# Create an image from the array as our `obj_array` is stored in an  numbered array form.
new_imageA = Image.fromarray(obj_array, mode="CMYK")

# Show the pixel value by the coordinate
print(new_imageA.getpixel((0,0)))

```
imported the numpy library from python

File: content/pillow/concepts/image/terms/fromarray/fromarray.md
Comment (SaviDahegaonkar) [inline content/pillow/concepts/image/terms/fromarray/fromarray.md:54]:
```suggestion
The result proves that the pixel at coordinate (0,0) is stored as `CMYK` combination colors with the values shown below.
```

File: content/pillow/concepts/image/terms/fromarray/fromarray.md
Comment (SaviDahegaonkar) [inline content/pillow/concepts/image/terms/fromarray/fromarray.md:63]:
```suggestion
The following Codebyte example demonstrates how to create a new image from the array and store it in an `RGB` color format.
```
I have corrected the sentence dramatically. Keep in note to always keep the sentence as readable as possible.

File: content/pillow/concepts/image/terms/fromarray/fromarray.md
Comment (SaviDahegaonkar) [inline content/pillow/concepts/image/terms/fromarray/fromarray.md:77]:
```suggestion
``` pseudo
import numpy as np
from PIL import Image

# Defined an array 3 by 3 with 200 as all the pixel values
obj_array = np.full((3, 3, 3), 200)

# Create an image from the array
new_imageB = Image.fromarray(obj_array, mode="RGB")

# Show the pixel value by the coordinate
print(new_imageB.getpixel((0,0)))
```
```
I have added one more channel for RGB image representation as the array should have three channels per pixel.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hello Author,
I have reviewed the given file, and made some changes, You can have a look at it and let me know.
Thanks,
Savi

**Comment by deenovita** [comment]
Thank you @SaviDahegaonkar, I have made some changes as requested

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hello @deenovita ,
There is a mistake you have committed this year.lock file, this file needs to be uncommitted by you, please look at the changes and let me know.
Thanks




<body>
<br class="Apple-interchange-newline"><table cellpadding="0" role="presentation" class="cf gJ" style="border-collapse: collapse; margin-top: 0px; width: auto; -webkit-font-smoothing: antialiased; font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: 0.875rem; letter-spacing: normal; display: block;"><tbody style="display: block;"><tr style="display: flex;"><td class="gF gK" style="margin: 0px; text-align: left; white-space: nowrap; padding: 0px; vertical-align: top; width: auto; line-height: 20px; display: block; max-height: 20px; color: rgb(119, 119, 119); font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><span translate="no" email="notifications@github.com" data-hovercard-id="notifications@github.com" name="Dewi Novita Sari" class="gD bco" role="gridcell" tabindex="-1" data-hovercard-owner-id="21" style="color: rgb(31, 31, 31); font-size: 0.875rem; font-weight: bold; white-space: nowrap; display: inline; vertical-align: top; padding-right: 3px; -webkit-font-smoothing: antialiased; font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; letter-spacing: normal; line-height: 20px;"><span style="position: relative; vertical-align: top;">Dewi Novita Sari</span></span><span class="cfXrwd"></span></td><br class="Apple-interchange-newline">
</body>
</html>

**Review by SaviDahegaonkar — APPROVED** [review]
Hello Author, 
I am approving this file and this ready for second review now.

**Comment by deenovita** [comment]
@SaviDahegaonkar Thank you!

**Comment by mamtawardhani** [comment]
Hey @ishg-153 please do ask the author to add an output image for the entry. Thanks! 😄

**Comment by ishg-153** [comment]
@mamtawardhani Thank you so much for pointing that out!

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@deenovita I have suggested some changes for you to work on. Please make those and let me know once you are done.

If you need any additional help please to ask.

**Comment by deenovita** [comment]
@ishg-153  I have made the requested changes. Please let me know if there are any other changes I need to make. Thank you!

**Review by ishg-153 — APPROVED** [review]
@deenovita Great! This entry is now good to merge. 

I have also made some minor changes from my end.

**Comment by ishg-153** [comment]
@deenovita Note that I won't be merging this right away, as we are waiting for another PR to be merged before moving forward with this one. 

Please don't delete this PRs' corresponding branch till it's merged. There's no other change required from your end.

**Comment by deenovita** [comment]
@ishg-153 Sure, thank you so much!

**Comment by ishg-153** [comment]
Merging

**Pull request merged**
